src/SFA.DAS.PR.Application.UnitTests/AccountProviders/Queries/GetAccountProviders/GetAccountProvidersQueryValidatorTests.cs

```diff
@@ -0,0 +1,26 @@
+﻿using FluentValidation.TestHelper;
+using SFA.DAS.PR.Application.AccountProviders.Queries.GetAccountProviders;
+
+namespace SFA.DAS.PR.Application.UnitTests.AccountProviders.Queries.GetAccountProviders
+{
```

Comment:
Class level namespace. Please check your VS settings and match it up as specified in the [WoW ](https://skillsfundingagency.atlassian.net/wiki/spaces/NDL/pages/3929210881/P1+Developers+Ways+of+working#Visual-studio-2022-settings)